web/src/features/auth/slices/authenticationSlice.ts

```diff
@@ -23,6 +24,7 @@ export const initialState: State = {
   isAuthenticated: false,
   tokenRefreshed: false,
   token: undefined,
+  idToken: undefined,
```

Comment:
is `token` for Keycloak in silver and `idToken` for gold Keycloak?

Reply:
Both are for gold keycloak. I believe the`idToken` existed in silver, but required as a parameter in the sign out URL for gold now.